config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
   root 'users#index'
   resources :users, only: [:index, :show] do
-  resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+    end
```

Comment:
- [ ] As per project requirement, add the route for like as well. [Hint: on line 6, `resources :likes, only: [:create]`]